packages/react-native-editor/ios/GutenbergDemo.xcodeproj/project.pbxproj

```diff
@@ -764,7 +764,7 @@
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
 				COPY_PHASE_STRIP = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
-				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "arm64 ";
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "";
```

Comment:
Looks like these changes were introduced in `1.70.3` but I don't recall any modification in the fix ([reference](https://github.com/WordPress/gutenberg/pull/38670)) related to this. @mchowning I'm wondering if you have any clue about where this change came from, thanks 🙇 .

Reply:
I don't. 🤔  It looks like @geriux made a related change [recently](https://github.com/WordPress/gutenberg/commit/323791acd67b1cdabaa67d48d8d4e1a077b2c23a). Perhaps he has some insight here.

Reply:
Before that, it actually looks like @fluiddot  😄  was the last person to add to add the "arm64" exclusion: https://github.com/WordPress/gutenberg/commit/a91013931d893f8778c9986713060111e1115745

Reply:
Oh, good catch, I didn't recall the change 😅. If the change was introduced in https://github.com/WordPress/gutenberg/commit/323791acd67b1cdabaa67d48d8d4e1a077b2c23a, I'm wondering why it's not in `trunk` 🤔. In any case, I think the change is valid as it's not expected that we exclude the `arm64` architecture for the simulator hence, I'll go ahead and merge the PR once the CI checks pass.